modules/apps/document-library/document-library-preview-audio-test/src/testIntegration/java/com/liferay/document/library/preview/audio/test/DLAudioFFMPEGAudioConverterTest.java

```diff
@@ -72,35 +73,42 @@ public void testDoesNotGenerateAudioPreview() throws Exception {
 			});
 	}
 
-	@ExpectedLogs(
-		expectedLogs = {
-			@ExpectedLog(
-				expectedLog = "ffmpeg", expectedType = ExpectedType.CONTAINS
-			),
-			@ExpectedLog(
-				expectedLog = "java.io.FileNotFoundException",
-				expectedType = ExpectedType.CONTAINS
-			),
-			@ExpectedLog(
-				expectedLog = "Unable to process",
-				expectedType = ExpectedType.CONTAINS
-			)
-		},
-		level = "ERROR", loggerClass = AudioProcessorImpl.class
-	)
 	@Test
 	public void testDoesNotGenerateAudioPreviewIfTheAudioIsCorrupt()
 		throws Exception {
 
-		_withDLAudioFFMPEGAudioConverterConfiguration(
-			true,
-			() -> {
-				FileEntry fileEntry = _createAudioFileEntry(
-					"audio_corrupt.wav");
-
-				Assert.assertFalse(
-					AudioProcessorUtil.hasAudio(fileEntry.getFileVersion()));
-			});
+		try (LogCapture logCapture = LoggerTestUtil.configureLog4JLogger(
+				"com.liferay.portlet.documentlibrary.util.AudioProcessorImpl",
+				LoggerTestUtil.ERROR)) {
+
+			_withDLAudioFFMPEGAudioConverterConfiguration(
+				true,
+				() -> {
+					FileEntry fileEntry = _createAudioFileEntry(
+						"audio_corrupt.wav");
+
+					Assert.assertFalse(
+						AudioProcessorUtil.hasAudio(
+							fileEntry.getFileVersion()));
+				});
+
+			List<LogEntry> logEntries = logCapture.getLogEntries();
+
+			Assert.assertTrue(
+				StringUtil.contains(
```

Comment:
@MrJohn1911 This StringUtil.contains() usage is too weird, please change it to plain String contains assert.